apps/examples/websocket/websocket_main.c

```diff
@@ -782,19 +777,23 @@ int websocket_main(int argc, char *argv[])
 			goto error_with_function_failure;
 		}
 		pthread_setname_np(tid, "websocket client");
-		pthread_detach(tid);
+		pthread_join(tid, &exit_value);
```

Comment:
You don't use a result of child pthread in yours.
Let's remove exit_value and fill NULL as a second argument at pthread_join.

Reply:
Agreed. Thanks